roles/edpm_multipathd/tasks/update.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2025 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure multipathd is running on the EDPM host
```

Comment:
So, we don't want to run `configure.yml` during minor update when moving to containers to host? if yes, then it's full deployment.  Creating update.yml and then including install.yml and run.yml looks like an antipattern to me.

Reply:
Ideally, we would just run any needed migration tasks, and any tasks needed to update. Given that, this looks ok to me. At some point, we could remove the migration related tasks (stopping the containers, etc) from the update.

Reply:
The only need for doing an update is, in fact, to handle migrating the service to the host. Eventually (probably a while in the future) we can drop the update task when we're sure old deployments that initially ran the service in a container have been updated.

In other words, the update tasks only need to run once on brownfield deployments.

That said, I will post a change to run the install and run tasks only when multipathd isn't  running on the host.